### PR TITLE
amp-consent: Improve console messages

### PR DESCRIPTION
**Pull request by Aerlinger**
Improves console warnings that were unclear or didn't provide adequate context. Also fixes minor spacing issues for the same.

### CODE REVIEW

**Comment by Aerlinger** [comment]
Updating CLA

File: extensions/amp-consent/0.1/amp-consent.js [hunk 1]
```diff
@@ -464,7 +464,7 @@ export class AmpConsent extends AMP.BaseElement {
         const keys = Object.keys(config['policy']);
         for (let i = 0; i < keys.length; i++) {
           if (keys[i] != 'default') {
-            user().warn(TAG, `policy ${keys[i]} is currently not supported` +
+            user().warn(TAG, `policy ${keys[i]} is currently not supported ` +
```
Comment (zhouyx) [inline extensions/amp-consent/0.1/amp-consent.js]: Can we revert the change here.

Comment (Aerlinger) [reply extensions/amp-consent/0.1/amp-consent.js]: Sure, can I ask what the issue is? Currently there's no space between "supported" and "and".

Comment (zhouyx) [reply extensions/amp-consent/0.1/amp-consent.js]: 😄 my bad. nvm..

**Comment by aghassemi** [comment]
@zhouyx @Aerlinger ping regarding CLA (still failing) and merging this PR. Thanks.

**Comment by nainar** [comment]
@Aerlinger ping regarding CLA and rebasing this PR. Please reopen this PR when you are ready to take this action. Will be closing this in the meanwhile.
